### PR TITLE
SONAR-7411 drop support for HTTPs Tomcat connector

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Comment by SonarTech** [comment]
#### Analysis summary
SonarQube analysis reported 2 issues:

Watch the comments in this conversation to review them.

#### 2 unreported issues

Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:

1. Project (org.sonarsource.sonarqube:sonar-process:target/generated-sources/protobuf/org/sonar/process/systeminfo/protobuf): MAJOR Add a 'package-info.java' file to document the 'protobuf' package [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
1. [TomcatStartupLogs.java#L31](https://github.com/SonarSource/sonarqube/blob/0a95c87b3a1161d7f49632e97a90910b12cfa57b/server/sonar-server/src/main/java/org/sonar/server/app/TomcatStartupLogs.java#L31): MAJOR Remove this unused "props" private field. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1068)

File: sonar-ws/src/main/java/org/sonarqube/ws/client/HttpConnector.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
   public static final int DEFAULT_READ_TIMEOUT_MILLISECONDS = 60_000;
 
   /**
-   * Base URL with trailing slash, for instance "https://localhost/sonarqube/".
+   * Base URL with trailing slash, for instance "http://localhost/sonarqube/".
```
Comment (simonbrandhof) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/HttpConnector.java]: `https` can be kept here.

**Comment by simonbrandhof** [comment]
LGTM

File: sonar-ws/src/main/java/org/sonarqube/ws/client/HttpConnector.java (hunk 1 above)
Comment (sns-seb) [inline sonar-ws/src/main/java/org/sonarqube/ws/client/HttpConnector.java]: true

**Comment by SonarTech** [comment]
#### Analysis summary
SonarQube analysis reported 2 issues

#### 2 unreported issues

Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:

1. org.sonarsource.sonarqube:sonar-process:target/generated-sources/protobuf/org/sonar/process/systeminfo/protobuf: MAJOR Add a 'package-info.java' file to document the 'protobuf' package [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
1. [TomcatStartupLogs.java#L31](https://github.com/SonarSource/sonarqube/blob/61600d457f61b22fc46897c70601595a46f8285b/server/sonar-server/src/main/java/org/sonar/server/app/TomcatStartupLogs.java#L31): MAJOR Remove this unused "props" private field. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1068)

**Pull request merged**